backend/cfg/cfgize.ml

```diff
@@ -459,6 +419,30 @@ let rec add_blocks :
       | Cfg.Tailcall _ | Cfg.Call_no_return _ ->
         body
     in
+    let can_raise =
+      (* Recompute [can_raise] and check that instruction in the middle
+         of the block do not raise, i.e., only the terminator, or the last
+         instruction (when the terminator is just a goto) can raise. *)
+      let last_can_raise =
+        match (terminator.Cfg.desc : Cfg.terminator) with
```

Comment:
Isn't it `can_raise_terminator`?

Reply:
No, it is checking if the terminator is just a goto. Maybe I should rename the variable.

Reply:
Silly me, I just read the variable name and thought
"surely we already have something to compute that"...